packages/components-library/tests/unit/filters/CheckboxFilter.spec.ts

```diff
@@ -81,4 +81,17 @@ describe('CheckboxFilter.vue', () => {
       expect(wrapper.findAll('input[type=checkbox]').length).toBe(1)
     })
   })
+
+  describe('SatisfyAllCheckbox', () => {
+    let wrapper: any
+    beforeEach(() => { wrapper = getWrapper({ }) })
```

Comment:
no need to pass empty object. just getWrapper() is enough (readability)

Reply:
Fixed in 1c81229